### PR TITLE
Doks installation guide

### PR DESCRIPTION
**Pull request by hubvu**
- added a new installation guide for installing Ondat onto DigitialOcean Kubernetes (DOKS)
- small/minor typo fixes to the rest of the installation guides
- consistent formatting and layout for the rest of the guides

### CODE REVIEW

File: docs/install/Anthos.md [hunk 1]
```diff
@@ -19,15 +19,15 @@ This guide will walk you through installing Ondat onto an [Anthos cluster](https
...
 
-> ⚠️ Make sure you have a running Anthos user cluster with a minimum of 3 worker nodes and the sufficient [Role-Based Access Control (RBAC)](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) permissions to deploy and manage applications in the cluster.
+> ⚠️ Make sure you have a running Google Anthos cluster with a minimum of 3 worker nodes and the sufficient [Role-Based Access Control (RBAC)](https://kubernetes.io/docs/reference/access-authn-authz/rbac/) permissions to deploy and manage applications in the cluster.
 
-> ⚠️ Make sure your Anthos clusters use [ubuntu_containerd](https://cloud.google.com/anthos/clusters/docs/on-prem/1.8/concepts/using-containerd) as the default node operating system. This version was chosen as the image includes the native Linux components required for Ondat.
+> ⚠️ Make sure your Google Anthos cluster use [`ubuntu_containerd`](https://cloud.google.com/anthos/clusters/docs/on-prem/1.8/concepts/using-containerd) as the default node operating system. This node operating system image has the required kernel modules available for Ondat to run successfully.
```
Comment (cannischan) [inline docs/install/Anthos.md:24]: would we need to change the version in the future?

File: docs/install/amazon-web-services-eks.md [hunk 2]
```diff
@@ -100,17 +100,17 @@ If you receive the message `No resources found` or see nodes marked as `NotReady
...
 kubectl storageos preflight
 ```
 
 ### Step 2 - Installing Ondat
 
-1. Define and export the `STORAGEOS_USERNAME` and `STORAGEOS_PASSWORD` environment variables that will be used to manage your Ondat instance. Set the `StorageClass` for etcd to use (this cannot be Ondat, as Ondat is dependent upon etcd). On AWS EKS, we suggest `gp3` for a good balance of performance and resilience or `io2` where top performance is essential.
-
-The default `StorageClass` in EKS is `gp2` which is not recommended, instead we will create a `gp3` `StorageClass` and set it as default, at least until we install Ondat:
+1. Define and export the `STORAGEOS_USERNAME` and `STORAGEOS_PASSWORD` environment variables that will be used to manage your Ondat instance. 
+2. Set the `StorageClass` for etcd to use (this cannot be Ondat, as Ondat is dependent upon etcd). On AWS EKS, we suggest `gp3` for a good balance of performance and resilience, or `io2` where top performance is essential.
```
Comment (cannischan) [inline docs/install/amazon-web-services-eks.md:112]: should we break out the "this cannot be ondat...." to make it a note with the emoji, like the rest of the paragraph ?

**Review by cannischan — COMMENTED** [review]
Had a look at this PR and made some comments :)

File: docs/install/Anthos.md (hunk 1 above)
Comment (hubvu) [reply docs/install/Anthos.md:24]: I don't believe so, unless something significantly changes with the ubuntu_containerd node os images causing the required kernel modules to be not available.

File: docs/install/Anthos.md [hunk 3]
```diff
@@ -19,15 +19,15 @@ This guide will walk you through installing Ondat onto an [Anthos cluster](https
...
 
 ## Procedure
 
 ### Step 1 - Conducting Preflight Checks
 
-* Run the following command to conduct preflight checks against the Anthos cluster to validate that Ondat prerequisites have been met before attempting an installation.
+- Run the following command to conduct preflight checks against the Anthos cluster to validate that Ondat prerequisites have been met before attempting an installation.
```
[the remark this answers is not in the document]
Comment (hubvu) [reply docs/install/Anthos.md]: yes - I have fixed this, thanks!

File: docs/install/amazon-web-services-eks.md [hunk 4]
```diff
@@ -36,7 +36,7 @@ aws ec2 describe-images \
...
+> In this example, we have used [eksctl](https://eksctl.io/introduction/) to create a cluster with 3 nodes of size `t3.large` running Ubuntu for EKS in the `eu-west-2` region. We have provided `100 GB` of disk space for each node. Note that by default, Ondat will store data locally in the node's file system under the path `/var/lib/storageos` on each node in [hyperconverged mode](/docs/concepts/nodes/#hyperconverged-mode). In a production infrastructure, we would create multiple Elastic Block Store (EBS) Volumes tweaked for performance or use ephemeral SSD storage and [mount our volumes under data device directories](/docs/concepts/volumes/) with some additions to user data. We would also implement some form of snapshots or backup of these underlying volumes to ensure continuity in a disaster scenario.
```
[the remark this answers is not in the document]
Comment (hubvu) [reply docs/install/amazon-web-services-eks.md]: fixed

File: docs/install/amazon-web-services-eks.md (hunk 2 above)
Comment (hubvu) [reply docs/install/amazon-web-services-eks.md]: yes - good idea - I have made the change

**Review by cannischan — APPROVED** [review]

**Pull request merged**